### PR TITLE
Support compliance policy v2 (#89)

### PR DESCRIPTION
**Pull request by ayalganem**
- Support compliance policy v2

### CODE REVIEW

File: dome9/resource_dome9_continuous_compliance_policy.go [hunk 1]
```diff
@@ -20,22 +20,17 @@ func resourceContinuousCompliancePolicy() *schema.Resource {
...
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
-			},
-			"cloud_account_type": {
+			"target_type": {
 				Type:         schema.TypeString,
 				Required:     true,
-				ValidateFunc: validation.StringInSlice([]string{"Azure", "Aws", "Google", "Kubernetes"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"Aws", "Azure", "Gcp", "Kubernetes", "OrganizationalUnit", "Terraform", "Generic"}, false),
```
Comment (erezweinstein5-zz) [inline dome9/resource_dome9_continuous_compliance_policy.go]: did we tested the "Terraform", "Generic" options?

File: website/docs/d/continuous_compliance_policy.html.markdown [hunk 2]
```diff
@@ -28,8 +28,8 @@ The following arguments are supported:
...
 In addition to all arguments above, the following attributes are exported:
 
-* `cloud_account_id` - GCP account name in Dome9.
-* `external_account_id` - The account number.
-* `cloud_account_type` - creation date for project in Google.
-* `bundle_id` - Organizational unit id.
+* `target_internal_id` - GCP account name in Dome9.
+* `target_external_id` - The account number.
+* `target_type` - creation date for project in Google.
+* `ruleset_id` - Organizational unit id.
```
Comment (erezweinstein5-zz) [inline website/docs/d/continuous_compliance_policy.html.markdown]: this entire section is wrong I think. what does Google has to do with this documentation?

**Review by erezweinstein5-zz — CHANGES_REQUESTED** [review]

File: dome9/resource_dome9_continuous_compliance_policy.go (hunk 1 above)
Comment (vasy100) [reply dome9/resource_dome9_continuous_compliance_policy.go]: No, but this is what the API can accept.

File: website/docs/d/continuous_compliance_policy.html.markdown (hunk 2 above)
Comment (vasy100) [reply website/docs/d/continuous_compliance_policy.html.markdown]: As an example, it doesn't matter which vendor it is.

**Review by erezweinstein5-zz — APPROVED** [review]

**Pull request merged**